cmd/restic/main.go

```diff
@@ -160,10 +187,16 @@ func main() {
 		exitCode = 10
 	case restic.IsAlreadyLocked(err):
 		exitCode = 11
+	case errors.Is(err, repository.ErrNoKeyFound):
+		exitCode = 12
```

Comment:
This is unrelated to main change, but I threw this in while I was here, since it is useful to programmatically detect a bad password.